windows/src/engine/keyman/UfrmKeyman7Main.pas

```diff
@@ -1948,36 +1948,33 @@ procedure TfrmKeyman7Main.CreateWnd;
 
 procedure TfrmKeyman7Main.StartKeymanX64;
 var
-  cmd: string;
-  si: TStartupInfoW;
-  pi: TProcessInformation;
-  v: PWideChar;
-  processHandle: THandle;
+  cmd, dir, params: string;
+  sei: TShellExecuteInfoW;
 begin
   if not IsWow64 then Exit;   // I4374
 
   try
-    FillChar(si, sizeof(TStartupInfoW), 0);
-    FillChar(pi, sizeof(TProcessInformation), 0);
-    si.cb := sizeof(TStartupInfo);
-    si.dwFlags := STARTF_USESHOWWINDOW;
-    si.wShowWindow := SW_HIDE;
-
-    if not DuplicateHandle(GetCurrentProcess, GetCurrentProcess,
-        GetCurrentProcess, @processHandle, 0, True, DUPLICATE_SAME_ACCESS) then
+    // TODO: use TKeymanPaths to find keymanx64?
```

Comment:
Shouldn't this TODO be removed since you didn't use TKeymanPaths?

Reply:
It's a low-priority TODO for the future 😁